voluncheer/opportunityboard/tests_views.py

```diff
@@ -91,33 +91,15 @@ def test_search(self):
         self.assertEqual(response.status_code, 200)
 
     def test_search_by_filters(self):
```

Comment:
Please add a docstring here to explain the test.

Reply:
Done